src/navigation/Navbar.jsx

```diff
@@ -7,16 +7,14 @@ import Avatar from "@mui/material/Avatar";
 import MenuItem from "@mui/material/MenuItem";
 import Menu from "@mui/material/Menu";
 import CircularProgress from "@mui/material/CircularProgress";
-import Button from "@mui/material/Button";
-import ProductDetails from "./productDetails";
-import { useState, useContext } from "react";
-import { AppContext } from "../App";
+import { useState } from "react";
 import { useQuery } from "react-query";
-import { getUserProfile } from "../apiCalls";
+import { getUserProfile } from "../axios/ApiCalls";
+import AddProduct from "../products/createProducts/AddProduct";
 
-export default function Navbar() {
+export default function Navbar({ setLoggedIn }) {
   const [anchorEl, setAnchorEl] = useState(null);
-  const { setLoggedIn, openModal, setOpenModal } = useContext(AppContext);
+  const [openModal, setOpenModal] = useState(false);
 
   const query = useQuery({
     queryKey: "getProfile",
```

Comment:
Here as well it should the key should be accessed from constants

Reply:
Okay, I'll make this update.